Helper/ShipmentOptions.php

```diff
@@ -217,22 +217,21 @@ public static function getValueFromAttribute(
      *
      * @return bool|null boolean value of the option named $key, or null when not set in $options
      */
-    public function getValueOfOptionWhenSet(string $key): ?bool
+    public static function getValueOfOptionWhenSet(string $key, $options): ?bool
```

Comment:
Zelfde 'probleem' als in deze https://github.com/myparcelnl/magento/pull/635 Wellicht hetzelfde maken, dan wordt het hopelijk ook geen merge conflict.